### DISCUSSION TITLE
Please add folia support.

### DISCUSSION THREAD

**Discussion by Potenza7** [GriefPrevention/GriefPrevention#2235] [General]
add folia support :/

**Comment by RoboMWM** [comment]
idk work for schedulers, but this would be at least after v17.

Anything else I'd do would be purely experimental and half-baked. I could hack together such a version just to do it but I'd not be spending much time bugfixing it, and definitely wouldn't be testing it myself.

anyone else is free to start legit discussions on this. I think such discussions exist, somewhat, from what I recall in the past, so you may be better off linking to that.

**Reply by Jikoo** [reply]
#2033

**Comment by Potenza7** [comment]
so its works with updated v17? you mean?

RoboMWM ***@***.***>, 5 Mar 2024 Sal, 19:50 tarihinde şunu
yazdı:

**Reply by Jikoo** [reply]
No. Robo said, "at least after v17." As GP is currently largely not thread-safe, Folia support would require a rewrite of a lot of the code. Modifying the scheduler to allow GP to enable would not make the rest of the code thread-safe, meaning every write is a roll of the dice for data corruption. It is realistically unlikely to be able to be done properly before the API breakage, which is v19, and because it's an optional feature tacked onto that it probably would not appear until later despite having to be factored into the initial design.

**Comment by Potenza7** [comment]
but still need claim plugin for folia do something

Adam ***@***.***>, 5 Mar 2024 Sal, 22:23 tarihinde şunu yazdı:

**Reply by RoboMWM** [reply]
![](https://tenor.com/view/fade-away-oooooooooooo-aga-emoji-crumble-gif-20008708.gif)

**Comment by ssquadteam** [comment]
Might want to look at https://github.com/ssquadteam/GriefPrevention-Folia
